fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -114,17 +117,24 @@ public class TableFunctionTable extends Table {
     public static final String PROPERTY_CSV_TRIM_SPACE = "csv.trim_space";
     public static final String PROPERTY_PARQUET_USE_LEGACY_ENCODING = "parquet.use_legacy_encoding";
 
+    private static final String PROPERTY_LIST_FILES_ONLY = "list_files_only";
+
     private String path;
     private String format;
-    private String compressionType;
+    private boolean listFilesOnly = false;
 
+    // for load data
     private int autoDetectSampleFiles;
     private int autoDetectSampleRows;
 
     private List<String> columnsFromPath = new ArrayList<>();
     private boolean strictMode = false;
     private final Map<String, String> properties;
 
+    private List<TBrokerFileStatus> fileStatuses = Lists.newArrayList();
```

Comment:
Is there a way to valid parameters if list_files_only is set? That is which parameters are needed, and which parameters mustn't be set.

Reply:
comments for load, unload, list files